pkg/volume/glusterfs/glusterfs_util.go

```diff
@@ -0,0 +1,55 @@
+package glusterfs
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+
+	"github.com/golang/glog"
+)
+
+// readGlusterLog will take the last 2 lines of the log file
+// on failure of gluster SetUp and return those so kubelet can
+// properly expose them
+// return nil on any failure
+func readGlusterLog(path string, podName string) error {
```

Comment:
@pmorie - I tested with hpcloud/tail and although it did tail the log it was a bit clunky in how it acted with our volumes, and it was pretty limited in overall functionality and seems overkill for what we are attempting to accomplish, and I would argue it's not worth integrating another dependency for what is hopefully a short term fix after mount.glusterfs does what we want.  Thoughts?